nerfstudio/process_data/colmap_utils.py

```diff
@@ -664,6 +664,9 @@ def create_ply_from_colmap(filename: str, recon_dir: Path, output_dir: Path):
 
     # Load point Positions
     points3D = torch.from_numpy(np.array([p.xyz for p in colmap_points.values()], dtype=np.float32))
+    # convert to nerfstudio coordinate system
+    points3D = points3D[..., torch.tensor([1, 0, 2])]
+    points3D[..., 2] *= -1
```

Comment:
Again this change is still not the right change, even though the training can go through. See https://github.com/nerfstudio-project/nerfstudio/pull/2793 for more discussions.